File: Calculater.py
while True: 
# Keep running the calculator until the user quits
    print("Welcome User!")
    print("-------------")
    #Instruction
    print("For addition select +")
    print("For subtraction select -")
    print("For multiplication select *")
    print("For division select /")
    a = int(input("Enter your first number "))
    o = (input("Select your operator "))
    b = int(input("Enter your second number "))
    if o == "+":
        print(a + b)
    elif o == "-":
        print(a - b)
    elif o == "*":
        print(a * b)
    elif o == "/":
        if b == 0:
            print("Can't divide by Zero(0)")
        else:
            print(a / b)
    else:
        print("Invalid Operator")

# Ask the user if they want to perform another calculation
    another_calculation = input("Do you want to perform another calculation? (yes/no): ")
    if another_calculation.lower() in ("no", "exit"):
        break  
# Exit the loop if the user enters "no" or "exit"

File: Calculater.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    while (true) {
      System.out.println("Enter Your First Number");
      double num1 = scanner.nextDouble();
      System.out.println("Select Your Operator (+, -, *, /, %)");
      System.out.println("Operators: \n +: Addition \n -: Subtraction \n *: Multiplication \n /: Division \n %: Modulus (remainder)");
      String operator = scanner.next();
      System.out.println("Enter Your Second Number");
      double num2 = scanner.nextDouble();

      if (operator.equals("+")) {
        System.out.println(num1 + num2);
      } else if (operator.equals("-")) {
        System.out.println(num1 - num2);
      } else if (operator.equals("*")) {
        System.out.println(num1 * num2);
      } else if (operator.equals("%")) {
        System.out.println(num1 % num2);
      } else if (operator.equals("/")) {
        if (num2 == 0) {
          System.out.println("Cannot Divide By Zero");
        } else {
          System.out.println(num1 / num2);
        }
      }

      System.out.print("Do you want to perform another calculation? (y/n): ");
      String continueChoice = scanner.next();

      if (continueChoice.equalsIgnoreCase("n")) {
        break;
      }
    }
    System.out.println("Exiting the calculator.");
  }
}
